ops/test/metabase.tf

```diff
@@ -14,4 +19,4 @@ module "metabase" {
 
   postgres_server_name = data.terraform_remote_state.persistent_test.outputs.postgres_server_name
   postgres_url         = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.metabase_db_uri.id})"
```

Comment:
As I understand it, this variable currently contains the server address, username, and password. Can we change it to just have the server address and name, which are things we already have and that aren't secrets?

Reply:
I don't know. Can we defer this as part of a greater key vault review?

Reply:
If the currently supplied values are successfully overriding the outdated ones here, then I think we should ship this, but open a separate issue for eliminating this specific secret—I don't know what the priority is, but I think waiting for a general sweep will just mean that we lose all context (plus I can't imagine when that will get prioritized).

Reply:
I'm happy to talk through this with you outside of a review. If we don't use this variable as the URL we can't connect through the private endpoint. It doesn't mean that we have to use this variable but we do have to replicate its config, as otherwise it'll try to connect publicly and fail.

Reply:
To be addressed here: https://app.zenhub.com/workspaces/prime-devops-606cb208a8c112000fd48349/issues/cdcgov/prime-devops/40